conda/cli/install.py

```diff
@@ -234,44 +234,6 @@ def install(args, parser, command='install'):
     ospecs = list(specs)
     add_defaults_to_specs(r, linked, specs, update=isupdate)
 
-    # Don't update packages that are already up-to-date
```

Comment:
This code really isn't necessary. If the solver returns the same set of packages as those already installed, the same message will be printed as was generated by this block. 

This block had one advantage: it bypassed the solver when it wasn't necessary. However, since the solver is tasked with identifying and possibly correcting corrupt environments, we don't _want_ to bypass the solver. And the solver is faster in these cases anyway.